shared/chat/conversation/messages/wrapper/unfurl/map/index.tsx

```diff
@@ -0,0 +1,145 @@
+import * as React from 'react'
+import * as Types from '../../../../../../constants/types/chat2'
+import * as Kb from '../../../../../../common-adapters/index'
+import * as Container from '../../../../../../util/container'
+import * as RouteTreeGen from '../../../../../../actions/route-tree-gen'
+import * as Styles from '../../../../../../styles'
+import {imgMaxWidth} from '../../../attachment/image/image-render'
+import {formatDurationForLocation} from '../../../../../../util/timestamp'
+import UnfurlImage from '../image'
+
+type Props = {
+  conversationIDKey: Types.ConversationIDKey
+  coord: Types.Coordinate
+  imageHeight: number
+  imageWidth: number
+  imageURL: string
+  isAuthor: boolean
+  liveLocationEndTime?: number | null
```

Comment:
Why not just `liveLocationEndTime?: number`

Reply:
Just matching an autogen'd type.

Reply:
i think the local types should be what we want them to be. i think its simpler if they're `foo?: string` so they're either a value or undefined and not value/undefined/null. when converting it from the rpc you can just do `|| undefined`